app/cdash/include/Messaging/Notification/Email/Mail.php

```diff
@@ -41,60 +34,9 @@ public function setRecipient($recipient)
         $this->recipient = $recipient;
     }
 
-    public function getDefaultSender(): string
-    {
-        if (!$this->defaultSender) {
-            $this->defaultSender = config('mail.from.address');
-        }
-        return $this->defaultSender;
-    }
-
-    public function getMailer(): Mailer
-    {
-        if (!$this->mailer) {
-            if (config('mail.driver') === 'smtp') {
```

Comment:
For context, here's where the bug actually was. `mail.driver` got renamed to `mail.default` when we upgraded to Laravel 9 in #1282